src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Lower.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 1999, 2020, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 1999, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I remained this line, because `Lower` was modified at 2021 but the copyright was not updated.

Reply:
Considering that this file is not related to this patch, I reverted this copyright.